etc/visual-studio/libopenthread-ncp-uart.vcxproj

```diff
@@ -73,4 +76,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
You might want a newline, but this is not how the files are generated from Visual Studio.